scripts/pip-post-compile.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
-bash scripts/_vendor/pip-post-compile.sh "$@"
```

Comment:
should this PR also remove this script from _vendor?

Reply:
it's ignored

https://github.com/dimagi/commcare-hq/blob/a7a1e0beb3da733b7634de123049953a4b845584/.gitignore#L9

Reply:
ah. that makes sense after looking at the old Makefile again. thanks for this change